--- onceki
+++ 2026-09-30T16-15-01Z
@@ -5,8 +5,7 @@
 T.C. TARIM VE ORMAN BAKANLIĞI
 SU YÖNETİMİ GENEL MÜDÜRLÜĞÜ
 -->
-×
-Ana Sayfa
+ANA SAYFA
 KURUMSAL YAPI
 GENEL MÜDÜR
 GENEL MÜDÜR YARDIMCILARI
@@ -36,16 +35,20 @@
 BASINDA BİZ
 HABER VİDEOLARI
 İLETİŞİM
--->
+‹
+MENÜ
+×
 Su Yönetimi Genel Müdürlüğü
+İÇME SUYU ARITMA TESİSLERİ PROJE ONAY GENELGESİ REVİZE EDİLDİ
 YEŞİLIRMAK HAVZASI SEKTÖREL SU TAHSİS PLANININ HAZIRLANMASI PROJESİ AÇILIŞ TOPLANTISI GERÇEKLEŞTİRİLDİ
 Türkiye Su Bilimi ve Yönetimi Dergisi’nin 2026 Yılı İlk Makalesi Yayımlandı
 KAYNAKTAN MUSLUĞA İÇME-KULLANMA SUYU MERCEK ALTINDA, YEREL İDARELERİN TEKNİK KAPASİTESİ ARTIRILIYOR
 KAYNAKTAN MUSLUĞA GÜVENLİ İÇME SUYU İÇİN BİRLİKTE ÇALIŞIYORUZ
-ARITILMIŞ KENTSEL ATIKSULARIN FARKLI ALTERNATİFLER İÇİN YENİDEN KULLANIMI
 Haber Arşivi
 Duyurular
 Tümü
+30.09.2026
+İÇME SUYU ARITMA TESİSLERİ PROJE ONAY GENELGESİ REVİZE EDİLDİ
 17.08.2026
 İŞKUR "İşgücü Uyum Programı" Sözlü Sınav Duyurusu
 03.04.2026
@@ -66,17 +69,23 @@
 Türkiye Su Atlası
 Neler Yapıyoruz?
 EĞİTİM - YAYIN PLATFORMLARIMIZ
-Rakamlarla Bakanlığımız
-Kırsal Kalkınma Hibe Desteği 2025 (Milyar ₺)
+Desteklemeler
+-->
+KIRSAL KALKINMA HİBE DESTEĞİ 2025
 14,48
-Korunan Alanlar (Milyon ha)
+Milyar ₺
+KORUNAN ALANLAR
 3,8
-Orman Alanı 2025 (Milyon ha)
+Milyon ha
+ORMAN ALANI 2025
 23,37
-Su Depolama Hacmi 2026 (Milyar m3)
+Milyon ha
+SU DEPOLAMA HACMİ 2026
 184
-Tarımsal Destekler 2026 (Milyar ₺)
+Milyar m3
+TARIMSAL DESTEKLER 2026
 167,6
+Milyar ₺
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
@@ -103,9 +112,9 @@
 Kamu Spotları
 SUYU VE GIDAYI KORU
 Kamu Spotları
-KAHVERENGİ_KOKARCA
+KAHVERENGİ KOKARCA
 Kamu Spotları
-ÇİFTÇİ_KAYIT_SİSTEMİ (ÇKS)
+ÇİFTÇİ KAYIT SİSTEMİ - (ÇKS)
 Kamu Spotları
 YANGINDA TEKNOLOJİ KULLANIMI
 Kamu Spotları
