--- onceki
+++ 2026-09-09T16-15-01Z
@@ -207,9 +207,6 @@
 9. Bölge Müdürlüğü Ankara Çamkoru Tabiat Parkı İhale İşletmecilik İhale İlanları ...
 DKMP
 25.08.2026
-TARDİP Projesi Hayvan Gölgeliği Yapımı İşi İhale İlanı
-BUGEM
-25.08.2026
 TARDİP Projesi Sac ve Beton Sıvat Yapımı ve Montajı İhale İlanı
 BUGEM
 18.08.2026
@@ -227,6 +224,9 @@
 24.07.2026
 9. Bölge Müdürlüğü Bolu Gölcük Tabiat Parkı Saha İşleri İhale İlanı
 DKMP
+24.07.2026
+10. Bölge Müdürlüğü Sinop İnaltı Mağarası Tabiat Parkı Yapım İşleri İhale İlanı
+DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
