--- onceki
+++ 2026-08-13T16-15-01Z
@@ -113,14 +113,14 @@
 Kamu Spotları
 BALIKÇILIK SEZON AÇILIŞI
 Tanıtım
+Bakanlık Tanıtım Videosu (Arapça)
+Tanıtım
+Bakanlık Tanıtım Videosu (İngilizce)
+Tanıtım
+Bakanlık Tanıtım Videosu
+Tanıtım
 Küre Dağları Milli Parkı Platin Yabanıl Sertifika Aldı
 TV Programları
-Bakanlığımız Tanıtım Videosu (Arapça)
-TV Programları
-Bakanlığımız Tanıtım Videosu (İngilizce)
-TV Programları
-Bakanlığımız Tanıtım Videosu
-TV Programları
 Orman Yangınlarına İlişkin Açıklama
 TV Programları
 Takmak Orman Yangın İlk Müdahale Ekip Binası Açılış Töreni
