--- onceki
+++ 2026-09-14T16-15-01Z
@@ -150,12 +150,12 @@
 BİLGİ EDİNME
 -->
 T.C. TARIM VE ORMAN BAKANLIĞI
+BAKAN YUMAKLI: “ŞIRNAK SİLOPİ SERA OTB EKONOMİYE YILLIK 1,2 MİLYAR LİRA KATKI SAĞLAYACAK”
 BAKAN YUMAKLI, KIRSAL KALKINMAYA 12,5 MİLYAR LİRALIK HİBE DESTEĞİ VERECEKLERİNİ DUYURDU
 4.1 MİLYAR LİRALIK DESTEKLEME ÖDEMESİ BUGÜN HESAPLARA AKTARILIYOR
 ÇEKİRDEKSİZ KURU ÜZÜM ALIM FİYATLARI BELİRLENDİ
 TÜRKİYE İLE ÇİN ARASINDA DEFNE YAPRAĞININ İHRACATINA İLİŞKİN PROTOKOL İMZALANDI
 BAKAN YUMAKLI “GÜBRE TEDARİKİNDE OLUŞABİLECEK RİSKLERE KARŞI KAYNAK ÇEŞİTLİLİĞİNİ ARTIRDIK”
-BİTKİSEL VE HAYVANSAL ÜRETİMDE DESTEK TUTARLARI ARTIRILDI
 Haber Arşivi
 ÇİFTÇİ
 TÜKETİCİ
@@ -165,6 +165,12 @@
 SU
 Duyurular
 Tümü
+14.09.2026
+Ünvan Değişikliği Sınavı Başarı Listesi
+PERGEM
+14.09.2026
+Tarım ve Orman Bakanlığı Sözleşmeli Gemi Adamı Uygulamalı Sınav İlanı
+PERGEM
 08.09.2026
 Türkiye İklim Akıllı ve Rekabetçi Büyüme Projesi – Zirai Mücadele Merkez Araştır ...
 TAGEM
@@ -189,14 +195,14 @@
 18.08.2026
 Fahri Av Müfettişlerine Yönelik Eğitim Programı
 4. Bölge Müdürlüğü
-13.08.2026
-Tarım ve Orman Bakanlığı Sözleşmeli Gemi Adamı Alım İlanı
-PERGEM
-12.08.2026
-Türkiye İklim Akıllı ve Rekabetçi Büyüme Projesi – Hizmet Alım İhalesi Sonuç Bil ...
-TAGEM
 İhale İlanları
 Tümü
+14.09.2026
+7. Bölge Müdürlüğü Osmaniye Çiftmazı Tabiat Parkı İşletmecilik İhale İlanı
+DKMP
+14.09.2026
+6. Bölge Müdürlüğü Isparta Hizmet Binası Bakım Onarım İhale İlanı
+DKMP
 11.09.2026
 “TUCSAP 1.2 TOB Dijital Modeli Uzaktan Algılama Faaliyetleri Kapsamında GPU Taba ...
 BTGM
@@ -221,12 +227,6 @@
 24.07.2026
 9. Bölge Müdürlüğü Bolu Gölcük Tabiat Parkı Saha İşleri İhale İlanı
 DKMP
-24.07.2026
-10. Bölge Müdürlüğü Sinop İnaltı Mağarası Tabiat Parkı Yapım İşleri İhale İlanı
-DKMP
-13.07.2026
-9. Bölge Müdürlüğü İhale İlanları
-DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
@@ -283,6 +283,14 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+#TarımSahada 9️⃣0️⃣. Buluşma 🤝
+Devamı
+11.09.2026
+TRGM
+29. ULUSLARARASI AGROTEC TARIM FUARI
+Devamı
+11.09.2026
+TRGM
 YEGFEST 2026 ANKARA'DA BAŞLADI
 Devamı
 08.09.2026
@@ -295,14 +303,6 @@
 Devamı
 10.09.2026
 TRGM
-SİLOPİ SERA OTB BÖLGE TARIMINA GÜÇ KATACAK
-Devamı
-07.09.2026
-TRGM
-#TarımSahada 8️⃣9️⃣. Buluşma 🤝
-Devamı
-04.09.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
