--- onceki
+++ 2026-07-24T16-00-01Z
@@ -140,21 +140,7 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
-23.07.2026 11.00
-Halkla İlişkiler Binası Komisyonlar Bloğu 1. Kat 3 No’lu Toplantı Salonu
-- Radyo ve Televizyon Üst Kurulu - Dijital Yayın Platformları - Film/Dizi Yapım Şirketleri
-Komisyonun 14. Toplantısı Toplantı Salonu Değişikliği
-DIŞİŞLERİ KOMİSYONU
-23.07.2026 13.00
-Halkla İlişkiler Binası 4. Kat 8 Nolu Toplantı Salonu
-“Yurtdışı Türkler ve Akraba Topluluklar Başkanlığı Alt Komisyon Tolantısı
-Dışişleri Bakanlığı Konsolosluk Hizmetleri ve Yurtdışında Yaşayan Vatandaşlar Genel Müdür Vekili Büyükelçi Sayın Esin ÇAKIL'ın “Güncel Duruma İlişkin Sunumları"
-DİLEKÇE KOMİSYONU
-23.07.2026 14.30
-Halkla İlişkiler Binası 4.Kat 7 Numaralı Komisyon Toplantı Salonu
-10. Genel Kurul Toplantısı
-İlköğretim ile Ortaöğretim Kurumlarında Akran Zorbalığının Araştırılması ve Alınabilecek Önlemlerin Belirlenmesi Alt Komisyonu Raporu
+EKLENMİŞ KOMİSYON GÜNDEMLERİ BULUNMAMAKTADIR.
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
