--- onceki
+++ 2026-10-05T16-15-01Z
@@ -202,6 +202,9 @@
 TRGM
 İhale İlanları
 Tümü
+05.10.2026
+9. Bölge Müdürlüğü Ankara Kompakt Fotoğraf Makinesi ve Dürbün Satın Alımı İhale ...
+DKMP
 02.10.2026
 6. Bölge Müdürlüğü Isparta Gölcük Tabiat Parkı Mal Alımı İhale İlanı
 DKMP
@@ -229,9 +232,6 @@
 11.09.2026
 “TUCSAP 1.2 TOB Dijital Modeli Uzaktan Algılama Faaliyetleri Kapsamında GPU Taba ...
 BTGM
-07.09.2026
-6. Bölge Müdürlüğü Antalya Beydağları Sahil MP İşletmecilik İhale İlanı
-DKMP
 Desteklemeler
 Tümü
 24.06.2026
@@ -294,6 +294,10 @@
 167,6
 Milyar ₺
 BİRİM HABERLERİ
+Tarım Sahada 9️⃣3️⃣. Buluşma
+Devamı
+02.10.2026
+TRGM
 GIDA İSRAFI ÇALIŞMA GRUBU 12. TOPLANTISINI GERÇEKLEŞTİRDİ
 Devamı
 02.10.2026
@@ -310,10 +314,6 @@
 Devamı
 28.09.2026
 GKGM
-KARS VE BİNGÖL’DE DOĞA TEMELLİ TARIM DÖNEMİ BAŞLIYOR
-Devamı
-28.09.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
