--- onceki
+++ 2026-08-14T16-15-01Z
@@ -38,11 +38,11 @@
 İLETİŞİM
 -->
 Su Yönetimi Genel Müdürlüğü
+ULUSAL SU KURULUNUN 6'NCI TOPLANTISI, SAYIN BAKANIMIZ İBRAHİM YUMAKLI’NIN BAŞKANLIĞINDA YAPILDI
 SU KAYNAKLARIMIZIN GELECEĞİ İÇİN SAHADAYIZ
 TÜRKİYE’NİN SU GELECEĞİ İÇİN STRATEJİK ADIM; NEHİR HAVZA YÖNETİM PLANLARINI HAZIRLIYORUZ
 AKARÇAY HAVZASI SU VERİMLİLİĞİ PROJESİ İÇİN HAZIRLIKLAR BAŞLADI
 “SU VERİMLİLİĞİ SEFERBERLİĞİ YARIŞMASI ESER KATALOĞU” ERİŞİME AÇILDI
-Su Güvenliği, Savaşlar ve KBRN Tehditleri Karşısında Stratejik Bir Güvenlik Meselesi Haline Geliyor
 Haber Arşivi
 Duyurular
 Tümü
@@ -65,16 +65,16 @@
 Neler Yapıyoruz?
 EĞİTİM - YAYIN PLATFORMLARIMIZ
 Destek Ödemeleri
-Kırsal Kalkınma Hibe Desteği 2024 (Milyar ₺)
-8,6
-Milli Park
-50
-Orman Alanı 2024 (Milyon ha)
-23,36
-Su Depolama Hacmi 2025 (Milyar m3)
-183,4
-Tarımsal Destekler 2025 (Milyar ₺)
-135
+Kırsal Kalkınma Hibe Desteği 2025 (Milyar ₺)
+14,48
+Korunan Alanlar (Milyon ha)
+3,8
+Orman Alanı 2025 (Milyon ha)
+23,37
+Su Depolama Hacmi 2026 (Milyar m3)
+184
+Tarımsal Destekler 2026 (Milyar ₺)
+167,6
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
@@ -126,12 +126,6 @@
 Takmak Orman Yangın İlk Müdahale Ekip Binası Açılış Töreni
 TV Programları
 Büyükbaş Hayvan Güvenli Elektronik Küpe Takma Programı
-TV Programları
-29.03.2019 - Habertürk - Özel Röportaj
-TV Programları
-26.02.2019 - Anadolu Ajansı - Editör Masası
-TV Programları
-07.02.2019 - TRT Haber - Özel Röportaj
 E-Hizmetler
 satis@tarimorman.gov.tr
 sales@tarimorman.gov.tr
