--- onceki
+++ 2026-10-09T16-15-01Z
@@ -294,6 +294,10 @@
 167,6
 Milyar ₺
 BİRİM HABERLERİ
+GENEL MÜDÜRÜMÜZ UĞUR ERDEM’İN KATILIMIYLA İYİ TARIM UYGULAMALARI SERTİFİKALI TÜRK PAMUĞU İÇİN UPK–BCI EŞDEĞERLİK PROTOKOLÜ İMZALANDI
+Devamı
+06.10.2026
+BUGEM
 İklim, Fenolojik Durum Ve Sulama İzleme İstatistik Bülteni Yayınlandı
 Devamı
 25.08.2026
@@ -310,10 +314,6 @@
 Devamı
 29.09.2026
 GKGM
-ŞANLIURFA VE MARDİN’DE ÇKS TOPLANTILARI YAPILDI
-Devamı
-29.09.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
