--- onceki
+++ 2026-08-28T16-15-01Z
@@ -265,7 +265,6 @@
 Faaliyet Raporları
 Tarımsal Destek Hesaplamaları
 İddialar ve Gerçekler
-Gıda İsrafının Önlenmesi
 Çok Dilli Tarım Orman Sözlüğü
 IPARD / OTP AĞI
 Ülke Masaları
@@ -284,6 +283,14 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+Tarım Sahada 8️⃣8️⃣. Buluşma
+Devamı
+28.08.2026
+TRGM
+MOĞOLİSTAN’DA UNCCD COP 17 KAPSAMINDA GENEL MÜDÜRLÜĞÜMÜZCE YÜRÜTÜLEN MERA ÇALIŞMALARI ANLATILDI
+Devamı
+27.08.2026
+BUGEM
 TRABZON VE RİZE’DE SÖZLEŞMELİ ÜRETİM HAMLESİ
 Devamı
 26.08.2026
@@ -296,14 +303,6 @@
 Devamı
 21.08.2026
 TRGM
-SİNOP’TA KDAKP KAPSAMINDA 65 MİLYON TL HİBE DESTEĞİ
-Devamı
-21.08.2026
-TRGM
-ÇEVRE VE İKLİM DOSTU TARIM İÇİN ÖNEMLİ ADIM
-Devamı
-18.08.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
