--- onceki
+++ 2026-08-24T16-15-01Z
@@ -283,6 +283,10 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+Tarım Sahada 8️⃣7️⃣. Buluşma
+Devamı
+21.08.2026
+TRGM
 SİNOP’TA KDAKP KAPSAMINDA 65 MİLYON TL HİBE DESTEĞİ
 Devamı
 21.08.2026
@@ -299,10 +303,6 @@
 Devamı
 13.08.2026
 GKGM
-GENEL MÜDÜRLÜĞÜMÜZ EV SAHİPLİĞİNDE SAĞLIK VE MİLLİ EĞİTİM BAKANLIKLARININ KATILIMIYLA ÜÇ BAKANLIK BİR ARAYA GELDİ
-Devamı
-13.08.2026
-GKGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
