--- onceki
+++ 2026-07-29T16-15-01Z
@@ -140,7 +140,11 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-EKLENMİŞ KOMİSYON GÜNDEMLERİ BULUNMAMAKTADIR.
+ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
+30.07.2026 14.00
+Halkla İlişkiler Binası Komisyonlar Bloku 3. Kat 5 No’lu Toplantı Salonu
+- Prof. Dr. Tuncay DİLCİ, Sivas Cumhuriyet Üniversitesi Eğitim Bilimleri Fakültesi - Adem SOLAK, Şiddetle Mücadele Vakfı (HEGEM) Başkanı - Öğretmenler, öğrenciler ve veliler
+Komisyonun 15. Toplantısı
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
