--- onceki
+++ 2026-08-08T16-15-01Z
@@ -140,11 +140,7 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-ADALET KOMİSYONU
-07.08.2026 15.30
-Halkla İlişkiler Binası 1. Kat 3 Numaralı Toplantı Salonu
-Adalet Komisyonu Toplantısı
-2/3793 esas numaralı Milli Dayanışma ve Toplumsal Bütünleşmenin Güçlendirilmesine Dair Kanun Teklifi
+EKLENMİŞ KOMİSYON GÜNDEMLERİ BULUNMAMAKTADIR.
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
