--- onceki
+++ 2026-08-06T16-15-01Z
@@ -140,11 +140,6 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-MİLLİ SAVUNMA KOMİSYONU
-05.08.2026 18.00
-HİB 3. Kat 5 nolu toplantı salonu
-Milli Savunma Komisyonu toplantısı
-2/3789 esas numaralı Bazı Kanunlarda Değişiklik Yapılmasına Dair Kanun Teklifi
 ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
 06.08.2026 14.00
 Halkla İlişkiler Binası Komisyonlar Bloku 3. Kat 5 No’lu Toplantı Salonu
