--- onceki
+++ 2026-08-14T16-15-01Z
@@ -272,16 +272,16 @@
 Kırsal Kalkınma Yatırım Progra ...
 EĞİTİM - YAYIN PLATFORMLARIMIZ
 Destek Ödemeleri
-Kırsal Kalkınma Hibe Desteği 2024 (Milyar ₺)
-8,6
-Milli Park
-50
-Orman Alanı 2024 (Milyon ha)
-23,36
-Su Depolama Hacmi 2025 (Milyar m3)
-183,4
-Tarımsal Destekler 2025 (Milyar ₺)
-135
+Kırsal Kalkınma Hibe Desteği 2025 (Milyar ₺)
+14,48
+Korunan Alanlar (Milyon ha)
+3,8
+Orman Alanı 2025 (Milyon ha)
+23,37
+Su Depolama Hacmi 2026 (Milyar m3)
+184
+Tarımsal Destekler 2026 (Milyar ₺)
+167,6
 BİRİM HABERLERİ
 ULUSAL GIDA KODEKS KOMİSYONU 2026 YILI İKİNCİ TOPLANTISI GERÇEKLEŞTİRİLDİ
 Devamı
@@ -354,12 +354,6 @@
 Takmak Orman Yangın İlk Müdahale Ekip Binası Açılış Töreni
 TV Programları
 Büyükbaş Hayvan Güvenli Elektronik Küpe Takma Programı
-TV Programları
-29.03.2019 - Habertürk - Özel Röportaj
-TV Programları
-26.02.2019 - Anadolu Ajansı - Editör Masası
-TV Programları
-07.02.2019 - TRT Haber - Özel Röportaj
 E-Hizmetler
 satis@tarimorman.gov.tr
 sales@tarimorman.gov.tr
