--- onceki
+++ 2026-08-18T16-15-01Z
@@ -164,6 +164,12 @@
 SU
 Duyurular
 Tümü
+18.08.2026
+Bilimsel Görüş - Gıda Olarak Kullanılabilecek Bitkiler Komisyonu
+GKGM
+18.08.2026
+Fahri Av Müfettişlerine Yönelik Eğitim Programı
+4. Bölge Müdürlüğü
 13.08.2026
 Tarım ve Orman Bakanlığı Sözleşmeli Gemi Adamı Alım İlanı
 PERGEM
@@ -188,14 +194,14 @@
 27.07.2026
 Ünvan Değişikliği Sözlü Sınavı
 PERGEM
-24.07.2026
-Tarım ve Orman Bakanlığı Sözleşmeli Personel Alım İlanı
-PERGEM
-21.07.2026
-Tarım ve Orman Bakanlığı Tarım Reformu Genel Müdürlüğü Kontrolör Yardımcılığı Gi ...
-TRGM
 İhale İlanları
 Tümü
+18.08.2026
+15. Bölge Müdürlüğü Sivas Balıklıgöl Tabiat Parkı İşletmecilik İhale İlanı
+15. Bölge Müdürlüğü
+12.08.2026
+DKMP 6.BÖLGE MÜDÜRLÜĞÜNE BAĞLI ANTALYA DKMP MÜDÜRLÜĞÜ ETÜT PROJE HİZMET ALIMI İH ...
+6. Bölge Müdürlüğü
 10.08.2026
 BATMAN TARIMA DAYALI İHTİSAS ORGANİZE SANAYİ BÖLGESİ MÜTEŞEBBİS HEYET BAŞKANLIĞI ...
 TRGM
@@ -208,6 +214,9 @@
 31.07.2026
 Tarım Reformu Genel Müdürlüğü (Tucsap) Çiftçi Örgütlerinin Ortak Makine Parklar ...
 TRGM
+30.07.2026
+Antalya İncekum Tabiat Parkı İdare Binası ve Çok Amaçlı Hizmet Ünitesi Tadilatı ...
+6. Bölge Müdürlüğü
 24.07.2026
 9. Bölge Müdürlüğü Bolu Gölcük Tabiat Parkı Saha İşleri İhale İlanı
 DKMP
@@ -217,15 +226,6 @@
 20.07.2026
 Türkiye'de Depremden Etkilenen İllerde Tarım Sektörünün İyileşmesi Projesi (TARD ...
 TRGM
-16.07.2026
-İstanbul (Pendik), İzmir (Bornova) ve Konya Veteriner Kontrol Enstitü Müdürlükle ...
-HAYGEM
-13.07.2026
-9. Bölge Müdürlüğü İhale İlanları
-DKMP
-13.07.2026
-12. Bölge Müdürlüğü Trabzon Pick-Up Hizmet Alımı İhale İlanı
-DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
@@ -283,6 +283,10 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+ÇEVRE VE İKLİM DOSTU TARIM İÇİN ÖNEMLİ ADIM
+Devamı
+18.08.2026
+TRGM
 Tarım Sahada 8️⃣6️⃣. Buluşma
 Devamı
 14.08.2026
@@ -299,10 +303,6 @@
 Devamı
 12.08.2026
 TRGM
-B-REÇETE SİSTEMİ KAPSAMINDA BİLGİLENDİRME TOPLANTILARI GERÇEKLEŞTİRİLDİ
-Devamı
-12.08.2026
-GKGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
