--- onceki
+++ 2026-09-29T16-15-01Z
@@ -169,6 +169,9 @@
 SU
 Duyurular
 Tümü
+29.09.2026
+16. Uluslararası Tarım Orman ve İnsan Fotoğraf Yarışmasında Finale Kalan Fotoğra ...
+EYDB
 25.09.2026
 3713 Sayılı Terörle Mücadele Kanunu Gereği Bakanlığımıza Yerleştirilenlerden İst ...
 PERGEM
@@ -196,11 +199,11 @@
 07.09.2026
 Ünvan Değişikliği Sözlü Sınav Sonuçları
 PERGEM
-25.08.2026
-Türkiye İklim Akıllı ve Rekabetçi Büyüme Projesi – Su-Team Yapım İşi İhalesi
-TAGEM
 İhale İlanları
 Tümü
+29.09.2026
+“TUCSAP 1.2 TOB Dijital Modeli Uzaktan Algılama Faaliyetleri Kapsamında GPU Taba ...
+btgm
 18.09.2026
 15. Bölge Müdürlüğü Sivas Divriği Milli Parkı Yapım İşleri İhale İlanı
 DKMP
@@ -228,10 +231,7 @@
 18.08.2026
 15. Bölge Müdürlüğü Sivas Balıklıgöl Tabiat Parkı İşletmecilik İhale İlanı
 15. Bölge Müdürlüğü
-03.08.2026
-15. Bölge Müdürlüğü Malatya Kınalı Keklik Üretim İstasyonu İnşaat İşleri İhale İ ...
-DKMP
-Destek Ödemeleri
+Rakamlarla Bakanlığımız
 Tümü
 24.06.2026
 SERTİFİKALI TOHUM KULLANIM DESTEĞİ
@@ -275,7 +275,7 @@
 Dijital Tarım Kütüphanesi
 Kırsal Kalkınma Yatırım Progra ...
 EĞİTİM - YAYIN PLATFORMLARIMIZ
-Destek Ödemeleri
+Rakamlarla Bakanlığımız
 Kırsal Kalkınma Hibe Desteği 2025 (Milyar ₺)
 14,48
 Korunan Alanlar (Milyon ha)
@@ -287,6 +287,14 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+AVRUPA KOMİSYONU (DG SANTE) "ULUSAL KALINTI İZLEME PLANI" DENETİMİ AÇILIŞ TOPLANTISI GERÇEKLEŞTİRİLDİ
+Devamı
+29.09.2026
+GKGM
+ŞANLIURFA VE MARDİN’DE ÇKS TOPLANTILARI YAPILDI
+Devamı
+29.09.2026
+TRGM
 UN VE UNLU MAMUL İŞLETMELERİNDE HACCP TETKİK EĞİTİMİ KARAMAN’DA GERÇEKLEŞTİRİLDİ
 Devamı
 28.09.2026
@@ -299,14 +307,6 @@
 Devamı
 25.09.2026
 TRGM
-16. ULUSLARARASI TARIM ORMAN VE İNSAN FOTOĞRAF YARIŞMASI’NDA ÖDÜLLER SAHİPLERİNİ BULDU
-Devamı
-25.09.2026
-EYDB
-KAYSERİ’NİN KIRSAL KALKINMASINA YATIRIM: KDAKP 3. FAZ STRATEJİK PLANLARI DEĞERLENDİRİLDİ
-Devamı
-21.09.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
