--- onceki
+++ 2026-09-23T05-45-01Z
@@ -166,6 +166,9 @@
 SU
 Duyurular
 Tümü
+23.09.2026
+Yozgat Yerköy Jeotermal Kaynaklı Sera Tarıma Dayalı İhtisas Organize Sanayi Bölg ...
+TRGM
 22.09.2026
 İklim, Fenolojik Durum Ve Sulama İzleme İstatistik Bülteni
 TRGM
@@ -193,9 +196,6 @@
 19.08.2026
 Tarım ve Orman Bakanlığı 1851 4/B Sözleşmeli Personel Alımına İlişkin Duyuru
 PERGEM
-18.08.2026
-Fahri Av Müfettişlerine Yönelik Eğitim Programı
-4. Bölge Müdürlüğü
 İhale İlanları
 Tümü
 18.09.2026
