--- onceki
+++ 2026-10-08T16-15-01Z
@@ -155,12 +155,12 @@
 MENÜ
 ×
 T.C. TARIM VE ORMAN BAKANLIĞI
+BAKAN YUMAKLI, "DSİ VAKFI KÖY OKULLARINA YARDIM YÖRESEL ÜRÜNLER SERGİSİ"NİN AÇILIŞINI YAPTI
 TARIM VE ORMAN BAKANLIĞI GEÇEN AY 117 BİN 360 GIDA DENETİMİ YAPTI
 BAKAN YUMAKLI, TEKNOFEST GÜNEYDOĞU'YU ZİYARET ETTİ: SAVUNMA SANAYİSİNDEKİ MÜHENDİSLİK BAŞARISINI TARIM TEKNOLOJİLERİNDE DE YAKALAYACAĞIZ
 BAKANI YUMAKLI: “KAYNAKLARIMIZI KORUMAYA, YÖNETMEYE VE YARINLARA TAŞIMAYA DEVAM EDECEĞİZ”
 BAKAN YUMAKLI: “TÜRKİYE, 2005'TEN BU YANA DÜNYA UN İHRACATINDA BİRİNCİLİĞİNİ KORUYOR”
 BAKAN YUMAKLI 2028'DE TAMAMLANMASI HEDEFLENEN EDİRNE ÇÖMLEKKÖY BARAJI'NDA İNCELEMEDE BULUNDU
-BAKAN YUMAKLI: "GIDA ARZ GÜVENLİĞİ İÇİN TARIM DİPLOMASİSİ ŞART"
 Haber Arşivi
 ÇİFTÇİ
 TÜKETİCİ
@@ -202,6 +202,9 @@
 TRGM
 İhale İlanları
 Tümü
+08.10.2026
+TARDİP Projesi Ar-Ge Faaliyetleri İçin Deneme Biçerdöveri Temini İhale İlanı
+BUGEM
 05.10.2026
 9. Bölge Müdürlüğü Ankara Kompakt Fotoğraf Makinesi ve Dürbün Satın Alımı İhale ...
 DKMP
@@ -229,9 +232,6 @@
 14.09.2026
 6. Bölge Müdürlüğü Isparta Hizmet Binası Bakım Onarım İhale İlanı
 DKMP
-11.09.2026
-“TUCSAP 1.2 TOB Dijital Modeli Uzaktan Algılama Faaliyetleri Kapsamında GPU Taba ...
-BTGM
 Desteklemeler
 Tümü
 24.06.2026
@@ -294,6 +294,10 @@
 167,6
 Milyar ₺
 BİRİM HABERLERİ
+İklim, Fenolojik Durum Ve Sulama İzleme İstatistik Bülteni Yayınlandı
+Devamı
+25.08.2026
+TRGM
 Tarım Sahada 9️⃣3️⃣. Buluşma
 Devamı
 02.10.2026
@@ -310,10 +314,6 @@
 Devamı
 29.09.2026
 TRGM
-UN VE UNLU MAMUL İŞLETMELERİNDE HACCP TETKİK EĞİTİMİ KARAMAN’DA GERÇEKLEŞTİRİLDİ
-Devamı
-28.09.2026
-GKGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
