--- onceki
+++ 2026-10-01T16-15-01Z
@@ -143,7 +143,7 @@
 Aranacak Kelime
 Dönemi ve Yasama Yılı
 Son Dönem Tüm Yasama Yılları
-28.DÖNEM 5.Yasama Yılı ( 30.09.2026 - ve sonrası... )
+28.DÖNEM 5.Yasama Yılı ( 01.10.2026 - ve sonrası... )
 28.DÖNEM 4.Yasama Yılı ( 01.10.2025 - 30.09.2026 )
 28.DÖNEM 3.Yasama Yılı ( 01.10.2024 - 30.09.2025 )
 28.DÖNEM 2.Yasama Yılı ( 01.10.2023 - 30.09.2024 )
@@ -747,11 +747,11 @@
 İlk İmza Sahibi Olsun
 Son Durumu
 Tüm Durumlar
-İŞLEMDE
-KANUNLAŞTI
-GÜNDEMDE
 GERİ ALINDI (F)
+GÜNDEMDE
 KOMİSYONDA
+İŞLEMDE
+KANUNLAŞTI
 Esas Numarası
 Başlangıç Tarihi
 Bitiş Tarihi
