--- onceki
+++ 2026-09-19T05-45-01Z
@@ -190,11 +190,11 @@
 Tarım ve Orman Bakanlığı 1851 4/B Sözleşmeli Personel Alımına İlişkin Duyuru
 PERGEM
 18.08.2026
-Bilimsel Görüş - Gıda Olarak Kullanılabilecek Bitkiler Komisyonu
-GKGM
-18.08.2026
 Fahri Av Müfettişlerine Yönelik Eğitim Programı
 4. Bölge Müdürlüğü
+13.08.2026
+Tarım ve Orman Bakanlığı Sözleşmeli Gemi Adamı Alım İlanı
+PERGEM
 İhale İlanları
 Tümü
 18.09.2026
