--- onceki
+++ 2026-09-07T16-15-01Z
@@ -150,12 +150,12 @@
 BİLGİ EDİNME
 -->
 T.C. TARIM VE ORMAN BAKANLIĞI
+ÜRETİCİ VE YATIRIMCILARA IPARD-III PROGRAMI KAPSAMINDA AĞUSTOSTA 844,5 MİLYON LİRA ÖDENDİ
 450 MİLYON LİRALIK TARIMSAL DESTEKLEME ÖDEMESİ BUGÜN ÇİFTÇİLERİN HESAPLARINA AKTARILIYOR
 EMİNE ERDOĞAN, TÜRKİYE MİLLİ BOTANİK BAHÇESİ'Nİ ZİYARET ETTİ
 BAKAN YUMAKLI, SİVAS KONGRESİ'NİN 107. YIL DÖNÜMÜ TÖRENİNE KATILDI
 BAKAN YUMAKLI: TÜRKİYE 185 ÜLKEYE, 2 BİN 226 ÇEŞİT TARIM VE GIDA ÜRÜNÜ İHRAÇ EDİYOR
 SU ÜRÜNLERİ AV SEZONU BAKAN YUMAKLI’NIN KATILDIĞI TÖRENLE AÇILDI
-VAN GÖLÜ DEVEBOYNU YARIMADASI TÜRKİYE'NİN 54. MİLLİ PARKI OLDU
 Haber Arşivi
 ÇİFTÇİ
 TÜKETİCİ
@@ -165,6 +165,12 @@
 SU
 Duyurular
 Tümü
+07.09.2026
+Organize Tarım Bölgelerinde Ön Tahsis ve Arsa Tahsisi Yapılacak Parsellere İlişk ...
+TRGM
+07.09.2026
+Ünvan Değişikliği Sözlü Sınav Sonuçları
+PERGEM
 25.08.2026
 Türkiye İklim Akıllı ve Rekabetçi Büyüme Projesi – Su-Team Yapım İşi İhalesi
 TAGEM
@@ -189,14 +195,11 @@
 12.08.2026
 Tarımsal Destek Ödemesi İstatistik Bülteni Yayınlandı
 TRGM
-07.08.2026
-Organize Tarım Bölgelerinde Ön Tahsis ve Arsa Tahsisi Yapılacak Parsellere İlişk ...
-TRGM
-04.08.2026
-Tarım ve Orman Bakanlığı Tarım Reformu Genel Müdürlüğü Kontrolör Yardımcısı Atam ...
-PERGEM
 İhale İlanları
 Tümü
+07.09.2026
+6. Bölge Müdürlüğü Antalya Beydağları Sahil MP İşletmecilik İhale İlanı
+DKMP
 26.08.2026
 15. Bölge Müdürlüğü Sivas Gökpınar Gölü Tabiat Parkı Çevre Düzenleme İşleri İhal ...
 DKMP
@@ -224,9 +227,6 @@
 24.07.2026
 9. Bölge Müdürlüğü Bolu Gölcük Tabiat Parkı Saha İşleri İhale İlanı
 DKMP
-24.07.2026
-10. Bölge Müdürlüğü Sinop İnaltı Mağarası Tabiat Parkı Yapım İşleri İhale İlanı
-DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
@@ -283,6 +283,14 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+SİLOPİ SERA OTB BÖLGE TARIMINA GÜÇ KATACAK
+Devamı
+07.09.2026
+TRGM
+#TarımSahada 8️⃣9️⃣. Buluşma 🤝
+Devamı
+04.09.2026
+TRGM
 KIRSAL KALKINMANIN YOL HARİTASI: OTP AĞI
 Devamı
 03.09.2026
@@ -295,14 +303,6 @@
 Devamı
 01.09.2026
 TRGM
-Tarım Sahada 8️⃣8️⃣. Buluşma
-Devamı
-28.08.2026
-TRGM
-MOĞOLİSTAN’DA UNCCD COP 17 KAPSAMINDA GENEL MÜDÜRLÜĞÜMÜZCE YÜRÜTÜLEN MERA ÇALIŞMALARI ANLATILDI
-Devamı
-27.08.2026
-BUGEM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
