--- onceki
+++ 2026-10-09T16-15-01Z
@@ -140,16 +140,21 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-KADIN ERKEK FIRSAT EŞİTLİĞİ KOMİSYONU
-08.10.2026 13.30
-HALKLA İLİŞKİLER BİNASI 3. KAT 6 NUMARALI TOPLANTI SALONU
-KEFEK 14. TOPLANTI GÜNDEMİ
-KOMİSYON BAŞKANVEKİLİ SEÇİMİ
 KAMU İKTİSADİ TEŞEBBÜSLERİ KOMİSYONU
 13.10.2026 14.30
 Halkla İlişkiler Binası Komisyonlar Bölümü 1. Kat 3 numaralı toplantı salonu
 Komisyon Toplantısı
 Denetim Toplantısı
+AVRUPA BİRLİĞİ UYUM KOMİSYONU
+13.10.2026 15.30
+Halkla İlişkiler Binası Komisyonlar Bloku 4. Kat 9 numaralı toplantı salonu
+Komisyon Toplantısı
+6 Numaralı Komisyon Toplantısı Gündemi
+PLAN VE BÜTÇE KOMİSYONU
+14.10.2026 10.00
+Plan ve Bütçe Komisyonu D-121 Numaralı Salon
+Tasfiyesine Karar Verilen Bazı Yatırım Fonları Hakkında Kanun Teklifi
+(2/3939-Esas)
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
