--- onceki
+++ 2026-08-07T16-15-01Z
@@ -140,11 +140,6 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
-06.08.2026 14.00
-Halkla İlişkiler Binası Komisyonlar Bloku 3. Kat 5 No’lu Toplantı Salonu
-- Komisyon Üyelerinin Komisyon Raporunda Yer Verilebilecek Hususlara Dair Önerilerinin Alınması
-Komisyonun 17. Toplantısı
 ADALET KOMİSYONU
 07.08.2026 15.30
 Halkla İlişkiler Binası 1. Kat 3 Numaralı Toplantı Salonu
