--- onceki
+++ 2026-08-21T05-45-01Z
@@ -211,9 +211,6 @@
 31.07.2026
 6. Bölge Müdürlüğü Antalya İncekum Tabiat Parkı Bina Tadilatı İhale İlanı
 DKMP
-31.07.2026
-Tarım Reformu Genel Müdürlüğü (Tucsap) Çiftçi Örgütlerinin Ortak Makine Parklar ...
-TRGM
 30.07.2026
 Antalya İncekum Tabiat Parkı İdare Binası ve Çok Amaçlı Hizmet Ünitesi Tadilatı ...
 6. Bölge Müdürlüğü
@@ -226,6 +223,9 @@
 20.07.2026
 Türkiye'de Depremden Etkilenen İllerde Tarım Sektörünün İyileşmesi Projesi (TARD ...
 TRGM
+16.07.2026
+İstanbul (Pendik), İzmir (Bornova) ve Konya Veteriner Kontrol Enstitü Müdürlükle ...
+HAYGEM
 Destek Ödemeleri
 Tümü
 24.06.2026
