--- onceki
+++ 2026-10-02T16-15-01Z
@@ -202,6 +202,12 @@
 TRGM
 İhale İlanları
 Tümü
+02.10.2026
+6. Bölge Müdürlüğü Isparta Gölcük Tabiat Parkı Mal Alımı İhale İlanı
+DKMP
+02.10.2026
+"TARDİP Projesi Ar-Ge Faaliyetleri İçin Biçerdöver Temini" İhale İlanı
+BUGEM
 30.09.2026
 2. Bölge Müdürlüğü Balıkesir Ayvalık Adaları TP İşletmecilik İhale İlanı
 DKMP
@@ -226,12 +232,6 @@
 07.09.2026
 6. Bölge Müdürlüğü Antalya Beydağları Sahil MP İşletmecilik İhale İlanı
 DKMP
-26.08.2026
-15. Bölge Müdürlüğü Sivas Gökpınar Gölü Tabiat Parkı Çevre Düzenleme İşleri İhal ...
-DKMP
-26.08.2026
-9. Bölge Müdürlüğü Ankara Çamkoru Tabiat Parkı İhale İşletmecilik İhale İlanları ...
-DKMP
 Desteklemeler
 Tümü
 24.06.2026
@@ -294,6 +294,10 @@
 167,6
 Milyar ₺
 BİRİM HABERLERİ
+GIDA İSRAFI ÇALIŞMA GRUBU 12. TOPLANTISINI GERÇEKLEŞTİRDİ
+Devamı
+02.10.2026
+GKGM
 AVRUPA KOMİSYONU (DG SANTE) "ULUSAL KALINTI İZLEME PLANI" DENETİMİ AÇILIŞ TOPLANTISI GERÇEKLEŞTİRİLDİ
 Devamı
 29.09.2026
@@ -310,10 +314,6 @@
 Devamı
 28.09.2026
 TRGM
-Tarım Sahada 9️⃣2️⃣. Buluşma 🤝
-Devamı
-25.09.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
