--- onceki
+++ 2026-08-11T05-45-01Z
@@ -163,9 +163,6 @@
 07.08.2026
 Organize Tarım Bölgelerinde Ön Tahsis ve Arsa Tahsisi Yapılacak Parsellere İlişk ...
 TRGM
-05.08.2026
-Balıkesir Gönen Tarıma Dayalı İhtisas (Jeotermal Kaynaklı Sera) Organize Sanayi ...
-TRGM
 04.08.2026
 Tarım ve Orman Bakanlığı Tarım Reformu Genel Müdürlüğü Kontrolör Yardımcısı Atam ...
 PERGEM
@@ -190,6 +187,9 @@
 17.07.2026
 2026 Yılı Dönem Tayinleri Kesin Yerleştirme Sonuçları
 PERGEM
+16.07.2026
+Organik Tarım Kontrolörü Eğitimi Duyurusu
+BUGEM
 İhale İlanları
 Tümü
 10.08.2026
