--- onceki
+++ 2026-07-22T16-00-01Z
@@ -140,16 +140,6 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-ADALET KOMİSYONU
-21.07.2026 15.00
-Halkla İlişkiler Binası 1. Kat 3 Numaralı Toplantı Salonu
-Adalet Komisyonu Toplantısı
-2/3771 esas numaralı Çocuk Koruma Kanunu ile Bazı Kanunlarda Değişiklik Yapılmasına Dair Kanun Teklifi
-ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
-21.07.2026 15.00
-Halkla İlişkiler Binası Komisyonlar Bloğu 3. Kat 5 No’lu Toplantı Salonu
-- Adalet Bakanlığı - Sağlık Bakanlığı - Ulaştırma ve Altyapı Bakanlığı - Gençlik ve Spor Bakanlığı - Aile ve Sosyal Hizmetler Bakanlığı - Diyanet İşleri Başkanlığı
-Komisyonun 12 . Toplantısı
 ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
 22.07.2026 14.00
 Halkla İlişkiler Binası Komisyonlar Bloğu 3. Kat 5 No’lu Toplantı Salonu
@@ -157,14 +147,14 @@
 Komisyonun 13 Toplantısı
 DİLEKÇE KOMİSYONU İLE İNSAN HAKLARINI İNCELEME KOMİSYONU ÜYELERİNDEN KURULU KARMA KOMİSYONU
 22.07.2026 14.30
-Halkla İlişkiler Binası 3.Kat 6 Numaralı Komisyon Toplantı Salonu
+Halkla İlişkiler Binası 1.Kat 3 Numaralı Komisyon Toplantı Salonu
 KDK 2025 Yıllık Raporu
 1- Başkanlık Sunuşları 2- Kamu Denetçiliği Kurumu 2025 Yıllık Raporu a) Alt Komisyon Raporunun Sunumu b) Alt Komisyon Raporunun Benimsenmesinin Oylanması c) Alt Komisyon Raporunun Görüşülmesi
 ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
 23.07.2026 11.00
-Halkla İlişkiler Binası Komisyonlar Bloğu 3. Kat 5 No’lu Toplantı Salonu
-- Radyo ve Televizyon Üst Kurulu - Dijital Yayın Platformlarının - Film/Dizi yapım şirketleri
-Komisyonun 14 Toplantısı
+Halkla İlişkiler Binası Komisyonlar Bloğu 1. Kat 3 No’lu Toplantı Salonu
+- Radyo ve Televizyon Üst Kurulu - Dijital Yayın Platformları - Film/Dizi Yapım Şirketleri
+Komisyonun 14. Toplantısı Toplantı Salonu Değişikliği
 DIŞİŞLERİ KOMİSYONU
 23.07.2026 13.00
 Halkla İlişkiler Binası 4. Kat 8 Nolu Toplantı Salonu
