--- onceki
+++ 2026-08-13T16-15-01Z
@@ -149,12 +149,12 @@
 BİLGİ EDİNME
 -->
 T.C. TARIM VE ORMAN BAKANLIĞI
+ULUSAL SU KURULUNUN 6'NCI TOPLANTISI, BAKAN YUMAKLI’NIN BAŞKANLIĞINDA YAPILDI
 BAKAN YUMAKLI: SİLVAN PROJESİ TAMAMLANDIĞINDA EKONOMİMİZE YILLIK 43 MİLYAR LİRALIK KATKI SAĞLAYACAK
 TARIM VE ORMAN BAKANI YUMAKLI İLE TİCARET BAKANI BOLAT’NIN KATILIMLARIYLA HUBUBAT SEKTÖRÜ İSTİŞARE TOPLANTISI GERÇEKLEŞTİRİLDİ
 BAKAN YUMAKLI: “ADANA KARATAŞ SERA ORGANİZE TARIM BÖLGESİ YILLIK 100 BİN TON ÜRETİM KAPASİTESİNE SAHİP OLACAK”
 81 İLDEKİ AİLE KAMPLARINA YOĞUN İLGİ
 688 MİLYON LİRALIK TARIMSAL DESTEKLEME ÖDEMESİ BUGÜN ÇİFTÇİLERİN HESAPLARINA AKTARILIYOR
-BAKAN YUMAKLI AÇIKLADI: 17 MİLYON BÜYÜKBAŞA DİJİTAL KÜPE GELİYOR
 Haber Arşivi
 ÇİFTÇİ
 TÜKETİCİ
@@ -164,6 +164,9 @@
 SU
 Duyurular
 Tümü
+13.08.2026
+Tarım ve Orman Bakanlığı Sözleşmeli Gemi Adamı Alım İlanı
+PERGEM
 12.08.2026
 Türkiye İklim Akıllı ve Rekabetçi Büyüme Projesi – Hizmet Alım İhalesi Sonuç Bil ...
 TAGEM
@@ -191,9 +194,6 @@
 21.07.2026
 Tarım ve Orman Bakanlığı Tarım Reformu Genel Müdürlüğü Kontrolör Yardımcılığı Gi ...
 TRGM
-17.07.2026
-Bitkisel Üretimde İyi Tarım Uygulamaları Kontrolör Eğitimi Katılımcı Listesi
-BUGEM
 İhale İlanları
 Tümü
 10.08.2026
@@ -283,6 +283,14 @@
 Tarımsal Destekler 2025 (Milyar ₺)
 135
 BİRİM HABERLERİ
+ULUSAL GIDA KODEKS KOMİSYONU 2026 YILI İKİNCİ TOPLANTISI GERÇEKLEŞTİRİLDİ
+Devamı
+13.08.2026
+GKGM
+GENEL MÜDÜRLÜĞÜMÜZ EV SAHİPLİĞİNDE SAĞLIK VE MİLLİ EĞİTİM BAKANLIKLARININ KATILIMIYLA ÜÇ BAKANLIK BİR ARAYA GELDİ
+Devamı
+13.08.2026
+GKGM
 KAYSERİ’DE TARIMSAL ÜRETİMİN GELECEĞİ YÜKSELİYOR
 Devamı
 12.08.2026
@@ -295,14 +303,6 @@
 Devamı
 07.08.2026
 TRGM
-TARIMSAL ÜRETİMİN PLANLANMASINDA YENİ YOL HARİTASI BELİRLENDİ
-Devamı
-06.08.2026
-TRGM
-AFET YÖNETİMİNDE DİJİTAL GÜÇ: DATS!
-Devamı
-05.08.2026
-TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
@@ -341,14 +341,14 @@
 Kamu Spotları
 BALIKÇILIK SEZON AÇILIŞI
 Tanıtım
+Bakanlık Tanıtım Videosu (Arapça)
+Tanıtım
+Bakanlık Tanıtım Videosu (İngilizce)
+Tanıtım
+Bakanlık Tanıtım Videosu
+Tanıtım
 Küre Dağları Milli Parkı Platin Yabanıl Sertifika Aldı
 TV Programları
-Bakanlığımız Tanıtım Videosu (Arapça)
-TV Programları
-Bakanlığımız Tanıtım Videosu (İngilizce)
-TV Programları
-Bakanlığımız Tanıtım Videosu
-TV Programları
 Orman Yangınlarına İlişkin Açıklama
 TV Programları
 Takmak Orman Yangın İlk Müdahale Ekip Binası Açılış Töreni
