--- onceki
+++ 2026-10-01T05-45-01Z
@@ -181,9 +181,6 @@
 24.09.2026
 “Devlet Destekli Tarım Sigortası Uygulamaları, I. Dönem: Ocak-Haziran 2026” İsta ...
 TRGM
-23.09.2026
-Yozgat Yerköy Jeotermal Kaynaklı Sera Tarıma Dayalı İhtisas Organize Sanayi Bölg ...
-TRGM
 22.09.2026
 İklim, Fenolojik Durum Ve Sulama İzleme İstatistik Bülteni
 TRGM
@@ -199,6 +196,9 @@
 07.09.2026
 Organize Tarım Bölgelerinde Ön Tahsis ve Arsa Tahsisi Yapılacak Parsellere İlişk ...
 TRGM
+07.09.2026
+Ünvan Değişikliği Sözlü Sınav Sonuçları
+PERGEM
 İhale İlanları
 Tümü
 30.09.2026
@@ -293,10 +293,6 @@
 167,6
 Milyar ₺
 BİRİM HABERLERİ
-NARENCİYE ÜRETİM MALİYETLERİNİN BELİRLENMESİNE YÖNELİK TOPLANTI GERÇEKLEŞTİRİLDİ
-Devamı
-30.09.2026
-TRGM
 AVRUPA KOMİSYONU (DG SANTE) "ULUSAL KALINTI İZLEME PLANI" DENETİMİ AÇILIŞ TOPLANTISI GERÇEKLEŞTİRİLDİ
 Devamı
 29.09.2026
@@ -313,6 +309,10 @@
 Devamı
 28.09.2026
 TRGM
+Tarım Sahada 9️⃣2️⃣. Buluşma 🤝
+Devamı
+25.09.2026
+TRGM
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
