--- onceki
+++ 2026-10-01T16-15-01Z
@@ -145,6 +145,11 @@
 Halkla İlişkiler Binası 1. Kat 3 No'lu Toplantı Salonu
 Komisyon Toplantısı
 Yabancı Dijital Konaklama Platformları Kanunu Teklifi
+GÜVENLİK VE İSTİHBARAT KOMİSYONU
+07.10.2026 15.00
+Halkla İlişkiler Binası Komisyonlar Bloğu 1. Kat 3 Nolu Toplantı Salonu
+GÜNDEM: 1. Başkanlık sunuşları, 2. Devlet İstihbarat Hizmetleri ile Güvenlik ve İstihbarî Nitelikteki Faaliyetlere İlişkin 2025 Yılı Raporunun görüşülmesi.
+25 Numaralı Toplantı Gündemi
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
