--- onceki
+++ 2026-07-23T16-00-01Z
@@ -141,16 +141,6 @@
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
 ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
-22.07.2026 14.00
-Halkla İlişkiler Binası Komisyonlar Bloğu 3. Kat 5 No’lu Toplantı Salonu
-- Kahramanmaraş ve Şanlıurfa Milletvekilleri
-Komisyonun 13 Toplantısı
-DİLEKÇE KOMİSYONU İLE İNSAN HAKLARINI İNCELEME KOMİSYONU ÜYELERİNDEN KURULU KARMA KOMİSYONU
-22.07.2026 14.30
-Halkla İlişkiler Binası 1.Kat 3 Numaralı Komisyon Toplantı Salonu
-KDK 2025 Yıllık Raporu
-1- Başkanlık Sunuşları 2- Kamu Denetçiliği Kurumu 2025 Yıllık Raporu a) Alt Komisyon Raporunun Sunumu b) Alt Komisyon Raporunun Benimsenmesinin Oylanması c) Alt Komisyon Raporunun Görüşülmesi
-ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
 23.07.2026 11.00
 Halkla İlişkiler Binası Komisyonlar Bloğu 1. Kat 3 No’lu Toplantı Salonu
 - Radyo ve Televizyon Üst Kurulu - Dijital Yayın Platformları - Film/Dizi Yapım Şirketleri
