--- onceki
+++ 2026-09-18T16-15-01Z
@@ -150,12 +150,12 @@
 BİLGİ EDİNME
 -->
 T.C. TARIM VE ORMAN BAKANLIĞI
+BAKAN YUMAKLI: “TÜRKİYE ÖRTÜ ALTI ÜRETİMİ DÜNYADA DÖRDÜNCÜ, AVRUPA'DA İSE İKİNCİ SIRADA”
+1.3 MİLYAR LİRALIK DESTEKLEME ÖDEMESİ BUGÜN HESAPLARA AKTARILIYOR
 TÜRKİYE İLE ALMANYA ARASINDA TARIM VE GIDA SEKTÖRLERİNDE İŞ BİRLİĞİ İÇİN ORTAK NİYET BEYANI İMZALANDI
 ET ÜRÜNLERİNE YÖNELİK YENİ STANDARTLAR BELİRLENDİ
 SÜTTE YENİ DÖNEM
 BAKAN YUMAKLI: “ŞIRNAK SİLOPİ SERA OTB EKONOMİYE YILLIK 1,2 MİLYAR LİRA KATKI SAĞLAYACAK”
-BAKAN YUMAKLI, KIRSAL KALKINMAYA 12,5 MİLYAR LİRALIK HİBE DESTEĞİ VERECEKLERİNİ DUYURDU
-4.1 MİLYAR LİRALIK DESTEKLEME ÖDEMESİ BUGÜN HESAPLARA AKTARILIYOR
 Haber Arşivi
 ÇİFTÇİ
 TÜKETİCİ
@@ -197,6 +197,9 @@
 4. Bölge Müdürlüğü
 İhale İlanları
 Tümü
+18.09.2026
+15. Bölge Müdürlüğü Sivas Divriği Milli Parkı Yapım İşleri İhale İlanı
+DKMP
 17.09.2026
 9. Bölge Müdürlüğü Ankara Çamkoru Tabiat Parkı İşletmecilik İhale İlanı
 DKMP
@@ -224,9 +227,6 @@
 03.08.2026
 15. Bölge Müdürlüğü Malatya Kınalı Keklik Üretim İstasyonu İnşaat İşleri İhale İ ...
 DKMP
-31.07.2026
-6. Bölge Müdürlüğü Antalya İncekum Tabiat Parkı Bina Tadilatı İhale İlanı
-DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
@@ -283,6 +283,10 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+14 İLDE, 17 ORGANİZE TARIM BÖLGESİNDE 690 YATIRIM ALANI YATIRIMCILARLA BULUŞUYOR
+Devamı
+18.09.2026
+TRGM
 FİDAN ÜRETİCİLERİ ALT BİRLİĞİ GENEL MÜDÜRLÜĞÜMÜZÜ ZİYARET ETTİ
 Devamı
 17.09.2026
@@ -299,10 +303,6 @@
 Devamı
 16.09.2026
 GKGM
-TARIM MESLEK LİSELERİ YENİDEN YAPILANDIRILIYOR
-Devamı
-16.09.2026
-EYDB
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
