--- onceki
+++ 2026-09-01T05-45-01Z
@@ -150,12 +150,12 @@
 BİLGİ EDİNME
 -->
 T.C. TARIM VE ORMAN BAKANLIĞI
+SU ÜRÜNLERİ AV SEZONU BAKAN YUMAKLI’NIN KATILDIĞI TÖRENLE AÇILDI
 VAN GÖLÜ DEVEBOYNU YARIMADASI TÜRKİYE'NİN 54. MİLLİ PARKI OLDU
 BAKAN YUMAKLI: “TÜRKİYE, DÜNYA İNCİR ÜRETİMİNDE YÜZDE 28'LİK ORANLA BİRİNCİ SIRADA”
 ÇİFTÇİLERİN HESAPLARINA 67,8 MİLYON LİRA TARIMSAL DESTEK ÖDEMESİ AKTARILIYOR
 BAKAN YUMAKLI: TÜRKİYE 83,2 MİLYAR DOLARLIK TARIMSAL HASILAYLA DÜNYADA 7'NCİ, AVRUPA'DA BİRİNCİ SIRAYA YÜKSELDİ
 195,6 MİLYON LİRALIK TARIMSAL DESTEKLEME ÖDEMESİ BUGÜN HESAPLARA AKTARILIYOR
-EDİRNE’NİN BEREKETLİ TOPRAKLARINA 2.7 MİLYAR LİRALIK DEV SULAMA YATIRIMI
 Haber Arşivi
 ÇİFTÇİ
 TÜKETİCİ
@@ -221,12 +221,12 @@
 31.07.2026
 6. Bölge Müdürlüğü Antalya İncekum Tabiat Parkı Bina Tadilatı İhale İlanı
 DKMP
-30.07.2026
-Antalya İncekum Tabiat Parkı İdare Binası ve Çok Amaçlı Hizmet Ünitesi Tadilatı ...
-6. Bölge Müdürlüğü
 24.07.2026
 9. Bölge Müdürlüğü Bolu Gölcük Tabiat Parkı Saha İşleri İhale İlanı
 DKMP
+24.07.2026
+10. Bölge Müdürlüğü Sinop İnaltı Mağarası Tabiat Parkı Yapım İşleri İhale İlanı
+DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
