--- onceki
+++ 2026-09-29T16-15-01Z
@@ -140,7 +140,11 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-EKLENMİŞ KOMİSYON GÜNDEMLERİ BULUNMAMAKTADIR.
+BAYINDIRLIK, İMAR, ULAŞTIRMA VE TURİZM KOMİSYONU
+06.10.2026 15.00
+Halkla İlişkiler Binası 1. Kat 3 No'lu Toplantı Salonu
+Komisyon Toplantısı
+Yabancı Dijital Konaklama Platformları Kanunu Teklifi
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
