--- onceki
+++ 2026-08-25T19-30-49Z
@@ -165,6 +165,12 @@
 SU
 Duyurular
 Tümü
+25.08.2026
+Türkiye İklim Akıllı ve Rekabetçi Büyüme Projesi – Su-Team Yapım İşi İhalesi Son ...
+TAGEM
+25.08.2026
+İklim, Fenolojik Durum Ve Sulama İzleme İstatistik Bülteni Yayınlandı
+TRGM
 19.08.2026
 Tarım ve Orman Bakanlığı 1851 4/B Sözleşmeli Personel Alımına İlişkin Duyuru
 PERGEM
@@ -189,12 +195,6 @@
 04.08.2026
 Tarım ve Orman Bakanlığı Tarım Reformu Genel Müdürlüğü Kontrolör Yardımcısı Atam ...
 PERGEM
-30.07.2026
-Organik Tarım Kontrolör Eğitimi Katılımcı Listesi
-BUGEM
-28.07.2026
-İklim, Fenolojik Durum Ve Sulama İzleme İstatistik Bülteni Yayınlandı
-TRGM
 İhale İlanları
 Tümü
 18.08.2026
