--- onceki
+++ 2026-08-26T05-45-01Z
@@ -203,9 +203,6 @@
 12.08.2026
 DKMP 6.BÖLGE MÜDÜRLÜĞÜNE BAĞLI ANTALYA DKMP MÜDÜRLÜĞÜ ETÜT PROJE HİZMET ALIMI İH ...
 6. Bölge Müdürlüğü
-10.08.2026
-BATMAN TARIMA DAYALI İHTİSAS ORGANİZE SANAYİ BÖLGESİ MÜTEŞEBBİS HEYET BAŞKANLIĞI ...
-TRGM
 03.08.2026
 15. Bölge Müdürlüğü Malatya Kınalı Keklik Üretim İstasyonu İnşaat İşleri İhale İ ...
 DKMP
@@ -227,6 +224,9 @@
 13.07.2026
 9. Bölge Müdürlüğü İhale İlanları
 DKMP
+13.07.2026
+12. Bölge Müdürlüğü Trabzon Pick-Up Hizmet Alımı İhale İlanı
+DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
