--- onceki
+++ 2026-10-08T16-15-01Z
@@ -140,11 +140,6 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-GÜVENLİK VE İSTİHBARAT KOMİSYONU
-07.10.2026 15.00
-Halkla İlişkiler Binası Komisyonlar Bloğu 1. Kat 3 Nolu Toplantı Salonu
-GÜNDEM: 1. Başkanlık sunuşları, 2. Devlet İstihbarat Hizmetleri ile Güvenlik ve İstihbarî Nitelikteki Faaliyetlere İlişkin 2025 Yılı Raporunun görüşülmesi.
-25 Numaralı Toplantı Gündemi
 KADIN ERKEK FIRSAT EŞİTLİĞİ KOMİSYONU
 08.10.2026 13.30
 HALKLA İLİŞKİLER BİNASI 3. KAT 6 NUMARALI TOPLANTI SALONU
