--- onceki
+++ 2026-09-17T16-15-01Z
@@ -197,6 +197,9 @@
 4. Bölge Müdürlüğü
 İhale İlanları
 Tümü
+17.09.2026
+9. Bölge Müdürlüğü Ankara Çamkoru Tabiat Parkı İşletmecilik İhale İlanı
+DKMP
 14.09.2026
 7. Bölge Müdürlüğü Osmaniye Çiftmazı Tabiat Parkı İşletmecilik İhale İlanı
 DKMP
@@ -224,9 +227,6 @@
 31.07.2026
 6. Bölge Müdürlüğü Antalya İncekum Tabiat Parkı Bina Tadilatı İhale İlanı
 DKMP
-24.07.2026
-9. Bölge Müdürlüğü Bolu Gölcük Tabiat Parkı Saha İşleri İhale İlanı
-DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
@@ -283,6 +283,14 @@
 Tarımsal Destekler 2026 (Milyar ₺)
 167,6
 BİRİM HABERLERİ
+FİDAN ÜRETİCİLERİ ALT BİRLİĞİ GENEL MÜDÜRLÜĞÜMÜZÜ ZİYARET ETTİ
+Devamı
+17.09.2026
+GKGM
+TÜRK GIDA KODEKSİ İÇME SÜTLERİ TEBLİĞİNDE DEĞİŞİKLİK YAPILDI
+Devamı
+17.09.2026
+GKGM
 TÜRK GIDA KODEKSİ ET, HAZIRLANMIŞ ET KARIŞIMLARI VE ET ÜRÜNLERİ TEBLİĞİ GÜNCELLENDİ
 Devamı
 16.09.2026
@@ -295,14 +303,6 @@
 Devamı
 16.09.2026
 EYDB
-GENEL MÜDÜRÜMÜZ SAYIN DR. OSMAN YILDIZ KAYSERİ’DE SAHA İNCELEMELERİNDE BULUNDU
-Devamı
-15.09.2026
-TRGM
-TARIM ORMAN AKADEMİ GÜÇLENİYOR, BÜYÜYOR
-Devamı
-15.09.2026
-EYDB
 Kamu Spotları
 Tanıtım Videoları
 TV Programları
