--- onceki
+++ 2026-08-04T16-15-01Z
@@ -145,6 +145,11 @@
 Halkla İlişkiler Binası Komisyonlar Bloku 3. Kat 5 No’lu Toplantı Salonu
 Sunumlar: - Kamu Denetçiliği Kurumu - Türkiye İnsan Hakları ve Eşitlik Kurumu - Kişisel Verileri Koruma Kurumu
 Komisyonun 16. Toplantısı
+MİLLİ SAVUNMA KOMİSYONU
+05.08.2026 18.00
+HİB 3. Kat 5 nolu toplantı salonu
+Milli Savunma Komisyonu toplantısı
+2/3789 esas numaralı Bazı Kanunlarda Değişiklik Yapılmasına Dair Kanun Teklifi
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
