--- onceki
+++ 2026-07-31T05-45-01Z
@@ -180,9 +180,6 @@
 17.07.2026
 2026 Yılı Dönem Tayinleri Kesin Yerleştirme Sonuçları
 PERGEM
-16.07.2026
-Organik Tarım Kontrolörü Eğitimi Duyurusu
-BUGEM
 03.07.2026
 Organize Tarım Bölgelerinde Ön Tahsis ve Arsa Tahsisi Yapılacak Parsellere İlişk ...
 TRGM
@@ -192,6 +189,9 @@
 03.07.2026
 Ünvan Değişikliği Sınavı Hakkında Duyuru
 PERGEM
+03.07.2026
+2026 Yılı Dönem Tayinleri Geçici Yerleştirme Sonuçları
+PERGEM
 İhale İlanları
 Tümü
 24.07.2026
