--- onceki
+++ 2026-08-22T05-45-01Z
@@ -220,12 +220,12 @@
 24.07.2026
 10. Bölge Müdürlüğü Sinop İnaltı Mağarası Tabiat Parkı Yapım İşleri İhale İlanı
 DKMP
-20.07.2026
-Türkiye'de Depremden Etkilenen İllerde Tarım Sektörünün İyileşmesi Projesi (TARD ...
-TRGM
 16.07.2026
 İstanbul (Pendik), İzmir (Bornova) ve Konya Veteriner Kontrol Enstitü Müdürlükle ...
 HAYGEM
+13.07.2026
+9. Bölge Müdürlüğü İhale İlanları
+DKMP
 Destek Ödemeleri
 Tümü
 24.06.2026
