--- onceki
+++ 2026-07-31T16-15-01Z
@@ -141,10 +141,10 @@
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
 ŞANLIURFA VE KAHRAMANMARAŞ'TA OKULLARDA MEYDANA GELEN OLAYLAR İLE ÇOCUKLARIN DİJİTAL ORTAMLARDA KARŞILAŞTIKLARI RİSKLER VE OLUMSUZ ETKİLERİN TÜM YÖNLERİYLE ELE ALINARAK ARAŞTIRILMASI, ÇÖZÜM ÖNERİLERİ GELİŞTİRİLMESİ VE BENZER OLAYLARIN ÖNLENMESİ İÇİN ALINMASI GEREKEN TEDBİRLERİN BELİRLENMESİ AMACIYLA KURULAN MECLİS ARAŞTIRMASI KOMİSYONU
-30.07.2026 14.00
+04.08.2026 15.00
 Halkla İlişkiler Binası Komisyonlar Bloku 3. Kat 5 No’lu Toplantı Salonu
-- Prof. Dr. Tuncay DİLCİ, Sivas Cumhuriyet Üniversitesi Eğitim Bilimleri Fakültesi - Adem SOLAK, Şiddetle Mücadele Vakfı (HEGEM) Başkanı - Öğretmenler, öğrenciler ve veliler
-Komisyonun 15. Toplantısı
+Sunumlar: - Kamu Denetçiliği Kurumu - Türkiye İnsan Hakları ve Eşitlik Kurumu - Kişisel Verileri Koruma Kurumu
+Komisyonun 16. Toplantısı
 --> TBMM TV
 PARLAMENTOLAR ARASI İLİŞKİLER
 KÜTÜPHANE VE ARŞİV
