--- onceki
+++ 2026-10-07T16-15-01Z
@@ -140,16 +140,6 @@
 Elektronik Kitaplar
 İstiklal Mahkemeleri Kataloğu
 KOMİSYON GÜNDEMLERİ
-BAYINDIRLIK, İMAR, ULAŞTIRMA VE TURİZM KOMİSYONU
-06.10.2026 15.00
-Halkla İlişkiler Binası 1. Kat 3 No'lu Toplantı Salonu
-Komisyon Toplantısı
-Yabancı Dijital Konaklama Platformları Kanunu Teklifi
-PLAN VE BÜTÇE KOMİSYONU
-06.10.2026 15.30
-Plan ve Bütçe Komisyonu D-121 Numaralı Salon
-1211 Sayılı Türkiye Cumhuriyet Merkez Bankası Kanununun 42’nci Maddesi Gereğince Merkez Bankasının Faaliyetlerine İlişkin Bilgilendirme Toplantısı
-1211 Sayılı Türkiye Cumhuriyet Merkez Bankası Kanununun 42’nci Maddesi Gereğince Merkez Bankasının Faaliyetlerine İlişkin Bilgilendirme Toplantısı
 GÜVENLİK VE İSTİHBARAT KOMİSYONU
 07.10.2026 15.00
 Halkla İlişkiler Binası Komisyonlar Bloğu 1. Kat 3 Nolu Toplantı Salonu
